soliket/bandpass.py

```diff
@@ -117,7 +180,11 @@ def must_provide(self, **requirements):
         # and requires some params to be computed
         # Assign those from Foreground
         if "bandint_freqs" in requirements:
-            self.freqs = requirements["bandint_freqs"]["freqs"]
+            self.bands = requirements["bandint_freqs"]["bands"]
+            self.exp_ch = [k.replace("_s0", "") for k in self.bands.keys() 
+                    if "_s0" in k]
+            # self.eff_freqs = [np.array(self.bands[k]['nu'])
```

Comment:
can these commented lines be removed?

Reply:
sure, I just left them there as a placeholder in case we want to make also `BandPass` as a class that can be called without passing the `bands` dictionary from `Foreground`. I can remove them.